src/h5web/dataset-visualizer/VisToolbar.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Vis } from './models';
+import HeatmapToolbar from '../visualizations/heatmap/HeatmapToolbar';
+
+interface Props {
+  vis?: Vis;
+}
+
+function VisToolbar(props: Props): JSX.Element {
+  const { vis } = props;
+
+  if (vis === Vis.Heatmap) {
```

Comment:
For now, as we have only one supported vis, I kept this simple logic. We can move to the `Record` once we add more visus.